packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -61,6 +60,7 @@ export class TSClient implements Generatable {
     this.genericsInfo = new GenericArgsInfo(this.dmmf)
   }
 
+  // eslint-disable-next-line @typescript-eslint/require-await -- must be async to match previous interface
```

Comment:
It is not a public class (it is not even possible to load it through any of the private modules), we can change the interface.